### ISSUE TITLE
Images not showing for students

### ISSUE THREAD

**Issue by bfriesenvcc** [catalyst/moodle-local_sitenotice#67]
I've created a site notice with an image and have discovered that the image isn't showing for students. 

The attached image shows the student view at the top, and my (Admin) view at the bottom. The image  actually only shows up for me when I create it the notice and for a little while later, but disappears the next day or a couple of days later, so I think that's a caching issue?

![sitenotice](https://github.com/catalyst/moodle-local_sitenotice/assets/8193384/48036e6f-0114-4182-b5d7-795a27eda571)

**Comment by dmitriim** [comment]
Hi @bfriesenvcc 
Please make sure you are on the latest version of the plugin. I have just tested it locally and it works ok for me as well as we have few clients that use this plugin without any complains. So I can't replicate the issue.

If it's a floating then most likely something is not working in your infrastructure rather than code. I'd recommend further investigation to make sure you can provide steps to replicate. 

I will leave this one open for new in case anyone else is having a similar issue.

**Comment by bfriesenvcc** [comment]
Hmm.. were you sure to try logging in as a specific student (and not just switch role to student)? When I provide a URL to an image located elsewhere on our Moodle site, it works. However if I upload the image within the plugin, it is stored here and that's when the problem arises: https://ourmoodle.ca/pluginfile.php/1/local_sitenotice/content/2/togetherallwellbeing1.png

**Comment by dmitriim** [comment]
Yes. Login as a random (using login as and direct login) student works for me. 

Would be nice to see exact steps to replicate with all details.

Also you can try debugging and see where it's broken for you. This method is responsible for serving files for the plugin https://github.com/catalyst/moodle-local_sitenotice/blob/MOODLE_39_STABLE/lib.php#L62

**Comment by bfriesenvcc** [comment]
I will keep playing around with it to see what I can figure out.
